intel_extension_for_transformers/llm/runtime/graph/core/layers/ip_fusion_ffn.cpp

```diff
@@ -511,20 +720,52 @@ void JblasGemmCompInt8(float* activation, jblas::storage::gemm::IWeightBase* w1p
   auto w1ptr_ = reinterpret_cast<typename Launcher_epi::PrologueB::StorageWeight*>(w1ptr);
   auto w2ptr_ = reinterpret_cast<typename Launcher::PrologueB::StorageWeight*>(w2ptr);
   auto w3ptr_ = reinterpret_cast<typename Launcher_mul::PrologueB::StorageWeight*>(w3ptr);
-  utils::GemmProblem gp1(1, seq, fmid, fin, w1ptr_->mBlockSize);
-  utils::GemmProblem gp2(1, seq, fout, fmid, w2ptr_->mBlockSize);
-  utils::GemmProblem gp3(1, seq, fmid, fin, w3ptr_->mBlockSize);
   static Launcher_epi kernel_epi;
   static Launcher_mul kernel_mul;
   static Launcher kernel;
-  auto quanA1 = kernel_epi.mProA.createStorage(seq, fin, w1ptr_->mBlockSize, w1ptr_->IsAsym());
-  quanA1.assign(reinterpret_cast<int8_t*>(workspace));
-  auto quanA2 = kernel.mProA.createStorage(seq, fmid, w2ptr_->mBlockSize, w2ptr_->IsAsym());
-  quanA2.assign(reinterpret_cast<int8_t*>(workspace));
-  typename Launcher_epi::Param args1{gp1, {activation, fin, &quanA1}, {w1ptr_}, epi_prama1};
-  typename Launcher::Param args2{gp2, {tmp2, fmid, &quanA2}, {w2ptr_}, epi_prama2};
-  typename Launcher_mul::Param args3{gp3, {activation, fin, &quanA1}, {w3ptr_}, {tmp2, tmp1, fmid, fmid}};
-  GemmRunWithA_ffn<Parallel>(&kernel_epi, &kernel_mul, &kernel, args1, args3, args2, th);
+  GetCPUDevice();
+  if (_cd->isHybrid()) {
```

Comment:
is it possible that P&E cores be opaque for operators?

Reply:
I mean only small changes to ```paralleler``` not gemm kernels

Reply:
Yes, I will hide it to Scheduler in next PR. Graph will not to ​identify the CPU type.